click/core.py

```diff
@@ -752,6 +752,8 @@ def __init__(self, name, context_settings=None, callback=None,
         #: should show up in the help page and execute.  Eager parameters
         #: will automatically be handled before non eager ones.
         self.params = params or []
+        if help and '\f' in help:
```

Comment:
Can you add a comment here, describing what this is for and why? Will be good to merge after that.

Thanks for the PR!